addons/binding/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVHandler.java

```diff
@@ -0,0 +1,179 @@
+                            || (cmd == ValloxMVBindingConstants.STATE_BOOST)) {
+                        logger.debug("Changing state to: {}", command.toString());
+                        // Open WebSocket
+                        vows_send.request(channelUID, command.toString());
+                        vows_send.request(null, null);
+                    }
+                } catch (NumberFormatException nfe) {
+                    // Other commands like refresh
+                    return;
+                }
+            } else if (channelUID.getId().equals(ValloxMVBindingConstants.CHANNEL_ONOFF)) {
+                if (command.toString() == "ON") {
+                    vows_send.request(channelUID, "0");
+                } else if (command.toString() == "OFF") {
+                    vows_send.request(channelUID, "5");
+                }
+            } else if (channelUID.getId().equals(ValloxMVBindingConstants.CHANNEL_EXTR_FAN_BALANCE_BASE)) {
+                QuantityType<Dimensionless> quantity = commandToQuantityType(command, SmartHomeUnits.PERCENT);
+                vows_send.request(channelUID, Integer.toString(quantity.intValue()));
+            } else if (channelUID.getId().equals(ValloxMVBindingConstants.CHANNEL_SUPP_FAN_BALANCE_BASE)) {
+                QuantityType<Dimensionless> quantity = commandToQuantityType(command, SmartHomeUnits.PERCENT);
+                vows_send.request(channelUID, Integer.toString(quantity.intValue()));
+            } else if (channelUID.getId().equals(ValloxMVBindingConstants.CHANNEL_HOME_SPEED_SETTING)) {
+                QuantityType<Dimensionless> quantity = commandToQuantityType(command, SmartHomeUnits.PERCENT);
+                vows_send.request(channelUID, Integer.toString(quantity.intValue()));
+            } else if (channelUID.getId().equals(ValloxMVBindingConstants.CHANNEL_HOME_AIR_TEMP_TARGET)) {
+                // Convert temperatur to milli degree Kelvin
+                QuantityType<Temperature> quantity = commandToQuantityType(command, SIUnits.CELSIUS)
+                        .toUnit(SmartHomeUnits.KELVIN);
+                int milliKelvin = quantity.multiply(new BigDecimal(100)).intValue();
+                vows_send.request(channelUID, Integer.toString(milliKelvin));
+            }
+        }
+    }
+
+    @SuppressWarnings("unchecked")
+    protected <U extends Quantity<U>> QuantityType<U> commandToQuantityType(Command command, Unit<U> defaultUnit) {
+        if (command instanceof QuantityType) {
+            return (QuantityType<U>) command;
+        }
+        return new QuantityType<U>(new BigDecimal(command.toString()), defaultUnit);
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("Start initializing!");
+
+        String ip = getConfigAs(ValloxMVConfig.class).getIp();
+        vows_send = new ValloxMVWebSocket(ValloxMVHandler.this, ip);
+
+        BigDecimal bdUpdateInterval = getConfigAs(ValloxMVConfig.class).getUpdateinterval();
```

Comment:
Maybe another type then a `BigDecimal` for your updateInterval (note the capital I) would make code here way easier and more reabable.